index.html

```diff
@@ -15,21 +15,168 @@
+        <span></span>
+        <div class="card card3">
+          <h2>Multi-Post Stories Gain+Glory</h2>
+            <ul>
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavScript</li>
+              <li>html</li>
+            </ul>
+            <a href="#button">See Project</a>
+        </div>
+      </div>
+      <div class="works-card">
+        <span></span>
+        <div class="card card4">
+          <h2>Multi-Post Stories Gain+Glory</h2>
+            <ul>
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavScript</li>
+              <li>html</li>
+            </ul>
+            <a href="#button">See Project</a>
+        </div>
+      </div>
+      <div class="works-card">
+        <span></span>
+        <div class="card card5">
+          <h2>Multi-Post Stories Gain+Glory</h2>
+            <ul>
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavScript</li>
+              <li>html</li>
+            </ul>
+            <a href="#button">See Project</a>
+        </div>
+      </div>
+      <div class="works-card">
+        <span></span>
+        <div class="card card6">
+          <h2>Multi-Post Stories Gain+Glory</h2>
+            <ul>
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavScript</li>
+              <li>html</li>
+            </ul>
+            <a href="#button">See Project</a>
+        </div>
+      </div>
+
+    </div>
+      
+  </section>
+
+  <section class="about">
+    <div class="about-title">
+      <h1>About me</h1>
+      <p>
+        I can help you build a product , feature or website Look through some of my work and experience! 
+        If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.
       </p>
-      <ul class="headline-icons">
-        <li><a href="#twitter" ><img src="./assets/icons/twitter.svg" alt="Twitter"></a></li>
-        <li><a href="#linkedIn" ><img src="./assets/icons/Linkedin_icon.svg" alt="LinkedIn"></a></li>
-        <li><a href="#medium" ><img src="./assets/icons/medium.svg" alt="Medium"></a></li>
-        <li><a href="#github" ><img src="./assets/icons/github.svg" alt="Github"></a></li>
-        <li><a href="#angellist" ><img src="./assets/icons/angellist.svg" alt="Angellist"></a></li>
-      </ul>
-      <div class="headline-dropdown">
-        <a href=""><img src="./assets/icons/Disabled.svg" alt="Dropdown"></a>
+      <a href="#about-button">Get My Resume</a>
+    </div>
+    <div class="about-cards">
+
+      <div class="about-card about1">     
+          <img src="./assets/icons/icon-languages.svg" alt="Language-icon">
+          <div class="about-card-title">  
+            <h2>Languages</h2>
+            <ul>
+              <li>JavScript</li>
+              <li>Ruby</li>
+              <li>Html</li>
+              <li>css</li>
+            </ul>
+          </div>
+          
+      </div>
```

Comment:
I am a bit confused as to the ul section since the image and h2 in lines 138 and 140 are not lists. Can you please clarify why you have suggested I use ul for those?

Reply:
Sorry for not being so clear and for the late reply, when you wrote I was not around... I meant to change the main div one line 137 into `ul` since it is that one representing one card and make the one on line 139 into a `li`. hope this makes sense now.

Reply:
I used i `div` because the card contains `img` and `h1` elements. I felt using a `ul` would result in extra styling.

Reply:
> Sorry for not being so clear and for the late reply, when you wrote I was not around... I meant to change the main div one line 137 into `ul` since it is that one representing one card and make the one on line 139 into a `li`. hope this makes sense now.

You should wrap them inside the `li` tag as required, remember, your card has to represent one `ul` tag, not a div

Reply:
> I used i `div` because the card contains `img` and `h1` elements. I felt using a `ul` would result in extra styling.

I approved the project because of my late reply but this is exactly what you should have done, make that div a `ul` and the child to `li`. since the hx and img can not be a direct child of a ul so wrapping them within a `li` will make it possible, kindly fix that as you move on to the next step. Thanks